Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -4989,6 +4989,8 @@ - (void)_cancelPendingAutocorrectionContextHandler
 
 - (void)requestAutocorrectionContextWithCompletionHandler:(void (^)(UIWKAutocorrectionContext *autocorrectionContext))completionHandler
 {
+    SetForScope requestAutocorrectionContextScope { _isRequestingAutocorrectionContext, YES };
```

Comment:
NIT: it's fine to have this at the top of this method, but i wonder if maybe we should move it a bit lower down to right before we actually `requestAutocorrectionContext` since technically everything above that is not actually requesting an autocorrection context 😅